playwright/support/mappings.ts

```diff
@@ -18,3 +18,22 @@ export const pages: { [name: string]: Pages } = {
     route: "/work/01a28167-8c8d-4141-a32f-718539d5c8a4?featured=949699",
   },
 };
+
+export const elements = {
+  "advanced search link": "[href='/advanced-search']",
+  "advanced search button": "#submit-button",
+  "search button": "#searchbar-button-search-bar",
+  "requestable checkbox": "text=Requestable",
+  "login button": "[value='Submit']",
+  "first login for options button": "text=Log in for options >> nth=0",
+  "government documents checkbox":
+    "//span[contains(text(), 'Show only US government documents')]",
+  "Latin language checkbox": "//label[contains(.,'Latin')]",
+  "year filter button": "#year-filter-button",
+  "first read online button": "text=Read Online >> nth=0",
+  "first request button":
+    "[href='https://www.nypl.org/research/collections/shared-collection-catalog/hold/request/b10715506-i13895605']",
+  "English language checkbox":
+    "//*[@id='languages-checkbox-group-0-wrapper']/label/span[1]",
```

Comment:
Is there potentially a better/less potentially flaky locator we could access here? (this is arguably the hardest part of these Playwright tests, and i know in many instances, there just isn't something better!)

For this one in particular, I'm thinking something like `//input[text()='English']` might work. Lmk what you find.

Reply:
Yes! I've been looking into optimizing these as well. I had changed some elements but not all of them. I'll also update the "Latin language checkbox" when I find something that works.

Reply:
@samanthaandrews I tried a few combinations (`input` looks to be more of a sibling to the text) and the two CSS locators that seem to work are `span:text('English')` and `[id='languages-checkbox-group-0-wrapper']`. Unless `span` and `id` are the flaky parts of the locator?